### ISSUE TITLE
VOC algorithm implementation questions

### ISSUE THREAD

**Issue by leonardp** [Sensirion/embedded-sgp#136]
Hello,

after reading "SGP40 – VOC Index for Experts" I still have open questions regarding the VOC algorithm. 
The sampling interval in the example code ([here](https://github.com/Sensirion/embedded-sgp/blob/master/sgp40_voc_index/sensirion_voc_algorithm.h#L51) and [here](https://github.com/Sensirion/embedded-sgp/blob/master/sgp40_voc_index/sgp40_voc_index_example_usage.c#L67)) and application note is always 1 second.
However, the application I am aiming towards calls for longer sampling intervals (because of power savings).

Looking at the comments in the code I am left with the following questions:
1. Are there restrictions on how long the sampling period can be while in the learning phase?
2. Does the sampling period have any effects on  device-to-device variations?
3. Are there any other possible caveats/pitfalls one should be aware of from application development point-of-view?

Any help would be greatly appreciated.

**Comment by sromion** [comment]
Also interested, I had the same thoughts at the time of my coding.
Target : SGP40

Regards,
Thanks

**Comment by leonardp** [comment]
@sromion I just now had a look at closed issues (https://github.com/Sensirion/embedded-sgp/issues/130 https://github.com/Sensirion/embedded-sgp/issues/134) that this question has already be answered.
tl;dr
The sampling rate always has to be 1Hz.

@MBjoern I hope it is ok to ping you directly.
Is it *technically* possible to tweak the parameters of the algorithm to enable higher sampling intervals?
If that is the case, are there plans by Sensirion to provide a set of parameters for selected (higher) sampling intervals?
If not feel free to close this issue.

Kind Regards

**Comment by MBjoern** [comment]
@leonardp Thank you for contancting us. 
I understand your issue regarding power consumption and the idea to work with a different sampling interval.
I strongly suggest to digg deeper here into the topic of MOX sensor technology.

The VOC algorithm and sensor (operation mode of the MOX sensor) are a fine balenced system, which took lot's of weeks to understand and tweak accordingly. Hence, to my knowledge, it is not as simple as changing a few parameter in the algorithm to change the sampling interval. In theory, it is indeed possible to work with sampling intervals of 2 or even 4 seconds, but this may require different operation modes, not sure though.

I recommend to read all the provided materials and app notes about the SGP40 component itself and the MOX sensor technology, and then test the setup using our Sensirion Evaluation Kit, to check, if the delivered signal still satisfies your use case of providing accurate VOC signals. After that, you can think about the tweaking of the VOC algorithm.

Best regards and good look.
Björn
